cpp/clang/readme.md

```diff
@@ -0,0 +1,16 @@
+# How to setup clang in your project
+To begin with, these configs are based on clang tools 17+ versions, so in order to work with it you should update your clang tools (For installation example in docker you can refer to [embedded-tools](https://github.com/emlid/embedded-tools/blob/pr-journal-and-filesystem/docker/Dockerfile)) project).
+
+Next, you should add code-quality as a submodule in the root of your project
+```bash
+git submodule add git@github.com:emlid/code-quality.git code-quality
+```
+Then copy `*-example.sh` files to your `tools` directory and change lines with paths to files that clang needs to check.
```

Comment:
I also really not a fan of this approach, maybe we can change `example` scripts to provide arguments with paths to files? So we could use it like `./clang-tidy-launch.sh <my_paths>` or with exclude argument as well?

Reply:
good point but regex header filter stopped me from doing it before. I will try to think once again